docs/content/integrations/airbyte.mdx

```diff
@@ -31,14 +31,60 @@ An [Airbyte connection](https://docs.airbyte.com/understanding-airbyte/connectio
 
 ---
 
-## Loading Airbyte asset definitions into Dagster
+## Step 1: Connecting to Airbyte
+
+The first step in using Airbyte with Dagster is to tell Dagster how to connect to your Airbyte instance using an Airbyte [resource](/concepts/resources). This resource contains information on where the Airbyte instance is located and any credentials needed to access it.
+
+```python startafter=start_define_instance endbefore=end_define_instance file=/integrations/airbyte/airbyte.py dedent=4
+from dagster_airbyte import airbyte_resource
+
+airbyte_instance = airbyte_resource.configured(
+    {
+        "host": "localhost",
+        "port": 8000,
+        # If using basic auth, include username and password:
+        "username": "airbyte",
+        "password": {"env": "AIRBYTE_PASSWORD"},
+    }
+)
+```
+
+If you're running Airbyte [locally using `docker-compose`](https://docs.airbyte.com/deploying-airbyte/local-deployment#setup--launch-airbyte), the `host` and `port` parameters should be set to `localhost` and `8000`, respectively. The default basic auth credentials are a username `airbyte` and password `password`.
+
+If you're hosting Airbyte externally, you'll need to provide a hostname where the Airbyte webapp and API are accssible, typically on port `80`. For more information on the configuration options available for the Airbyte resource, see the [API reference](/\_apidocs/libraries/dagster-airbyte#dagster_airbyte.airbyte_resource).
+
+---
+
+## Step 2: Loading Airbyte asset definitions into Dagster
 
 The easiest way to get started using Airbyte with Dagster is to have Dagster automatically generate asset defintions from your Airbyte project. This can be done in one of two ways:
 
-- [Load asset definitions from Airbyte YAML configuration files](#loading-airbyte-asset-definitions-from-yaml-config), generated by Airbyte's [Octavia CLI](https://github.com/airbytehq/airbyte/tree/master/octavia-cli#what-is-octavia-cli)
-- [Load asset definitions from an Airbyte instance](#loading-airbyte-asset-definitions-from-an-airbyte-instance) via API, at initialization time
+- Load asset definitions from an Airbyte instance via API, at initialization time
+- Load asset definitions from Airbyte YAML configuration files, generated by Airbyte's [Octavia CLI](https://github.com/airbytehq/airbyte/tree/master/octavia-cli#what-is-octavia-cli)
+
+You can also manually-build asset definitions on a per-connection basis.
+
+<TabGroup>
 
-You can also [manually-build asset definitions](#manually-building-airbyte-asset-definitions) on a per-connection basis.
+<TabItem name="Loading from Airbyte instance">
+
+### Loading Airbyte asset definitions from an Airbyte instance
```

Comment:
These headers are displayed on the table of contents, even when the tab is collapsed, which is kind of awkward. Can we suppress them from showing, or remove them altogether?

Reply:
> more of a landing page, quickly explaining the value of Airbyte, and then linking out to some guides on how to get started

Yeah, I think this is the way to go eventually once we have more content. For now I bias to keeping it as a single page - I'm a little wary of contentless landing pages until we have a decent amount of sub-pages bc it pollutes docs search and adds clicks to find what a user is looking for.

> a quick example of setting a downstream dependency of an airbyte asset

#10217 adds a more simple example of that - dbt should probably go in its own guide page since it's more involved.

Reply:
I agree with @rexledesma on this one:

> These headers are displayed on the table of contents, even when the tab is collapsed, which is kind of awkward. Can we suppress them from showing, or remove them altogether?

Additionally, the anchors in non-visible tabs don't work correctly if you try to navigate directly to them. I'd expect to be taken right to the heading and for the appropriate tab to be open, but instead I'm taken to the title of the page. Clicking the links in the sidenav also doesn't do anything, which makes things appear broken.

I'd opt for removing the headings from the tabs for now and deciding whether we want to fix this behavior. I do think there's value in it, but I'm not sure it's the best use of time atm.

I'm not sure we need headings in the tabs if the idea is to make the section topic obvious, as the topic is also in the tab name. We could instead opt for styling these in bold or creating a `<span>` tag that stands out, for use in cases like this one.

Reply:
https://github.com/dagster-io/dagster/pull/10231 should fix that behavior (links to headings in tabs will switch to displaying that tab), which should make it easier for us to link to particular tabs etc

not sure if we have an opinion on whether or not those headers should still show in the sidebar - https://github.com/dagster-io/dagster/pull/10231 leaves them